kubecost/templates/frontend/frontend-configmap.yaml

```
@@ -124,7 +124,11 @@ data:
         location = /custom_504.html {
             internal;
         }
-
+        location /healthz {
+            add_header 'Content-Type' 'text/plain';
+            return 200 "healthy\n";
+            access_log off;
+        }
```

```suggestion
        # Needs to be served outside of the auth middleware
        location /healthz {
            add_header 'Content-Type' 'text/plain';
            return 200 "healthy\n";
            access_log off;
        }

```

Recommend keeping the comment. But overall this looks good!

---

We don't need the comment, it is outside the scope of middleware.